playbook/createGame.ts

```diff
@@ -0,0 +1,79 @@
+import { task } from 'hardhat/config';
+import { RankifyDiamondInstance, Rankify } from '../types';
+import { IRankifyInstanceCommons } from '../types/src/facets/RankifyInstanceMainFacet';
+
+task('createGame', 'Create new game')
+  .addOptionalParam('gameCreator', 'Player address who will create game')
+  .setAction(
+    async ({
+      gameCreator,
+    }: { gameCreator: string;},
+     hre) => {
```

Comment:
```
import { HardhatRuntimeEnvironment } from 'hardhat/types';
hre: HardhatRuntimeEnvironment
```

Reply:
Run pnpm install --frozen-lockfile
 ERR_PNPM_BROKEN_LOCKFILE  The lockfile at "/home/runner/work/contracts/contracts/pnpm-lock.yaml" is broken: Cannot convert undefined or null to object